src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -232,19 +244,24 @@ details.sd-dropdown {
     }
 
     @include legacy-backdrop-placeholder;
+
     background-color: var(--pst-sd-dropdown-bg-color) !important;
 
     // Add a left border with the same structure as our admonitions
     border-left: 0.2rem solid var(--pst-sd-dropdown-color) !important;
+
+    // stylelint-disable-next-line no-duplicate-selectors
```

Comment:
The linter complained about duplicate selectors, but in this case it didn't look safe to me to move the rules above (see line 223 and line 235)